src/components/navigation/header/Header.tsx

```diff
@@ -68,60 +74,109 @@ export const Header = ({
+                </div>
               )}
+              {renderPageLinks(links, false, pathname)}
+              {renderPageCTAs(ctas, false)}
+              <div className={styles.languageSwitcher}>
+                {defaultLanguage && (
+                  <LanguageSwitcher
+                    currentLanguage={currentLanguage}
+                    pathTranslations={pathTranslations}
+                  />
+                )}
+              </div>
+              <button
+                aria-haspopup="true"
+                aria-controls={sidebarID}
+                className={isOpen ? styles.open : styles.closed}
+                aria-expanded={isOpen}
+                onClick={toggleMenu}
+              />
             </div>
-          )}
-        </nav>
-      </header>
-    </FocusOn>
+            {isOpen && (
+              <div
+                className={styles.mobileMenu}
+                id={sidebarID}
+                aria-label="Mobile Menu"
+                onClick={() => setIsOpen(false)}
+              >
+                {renderPageLinks(sidebarLinks, true, pathname)}
+                {renderPageCTAs(sidebarCtas, true)}
+                {defaultLanguage && (
+                  <LanguageSwitcher
+                    currentLanguage={currentLanguage}
+                    pathTranslations={pathTranslations}
+                  />
+                )}
+              </div>
+            )}
+          </nav>
+        </header>
+      </FocusOn>
+      {pathname !== "/" && pathname !== "/" + currentLanguage && (
+        <BreadCrumb currentLanguage={currentLanguage} pathname={pathname} />
+      )}
+    </>
+  );
+};
+
+export const BreadCrumb = ({ currentLanguage, pathname }: BreadCrumbProps) => {
+  return (
+    <ul className={styles.breadCrumbMenu}>
+      {/* TODO: "Hjem" should be updated with translation */}
+      {[
+        "Hjem",
+        ...(pathname.includes("/" + currentLanguage + "/")
+          ? pathname.split("/").slice(2)
+          : pathname.split("/").slice(1)),
+      ].map((e, index, path) => {
+        const href =
+          "/" + currentLanguage + "/" + path.slice(1, index + 1).join("/");
+        const isLast = index === path.length - 1;
+
+        return (
+          <li key={index} className={styles.breadCrumb}>
+            <Link className={styles.breadCrumb} href={href}>
+              <Text
+                className={
+                  isLast ? styles.breadCrumbText : styles.breadCrumbLink
+                }
+                type={isLast ? "labelSemibold" : "desktopLink"}
+              >
+                {e.charAt(0).toUpperCase() + e.slice(1).toLowerCase()}
+              </Text>
+            </Link>
+            {!isLast && (
+              <span
+                className={
+                  index < path.length - 2
+                    ? styles.dotSeparator
+                    : styles.lastDotSeparator
+                }
+              >
+                •
+              </span>
+            )}
```

Comment:
Maybe place the dot separator before the current slug, instead of after the previous slug? Probably doesn't matter, but I think it makes more sense that the highlighted dot belongs to the last list item.

Reply:
Done